.github/workflows/on_pull_request.yml

```diff
@@ -27,17 +27,17 @@ jobs:
     outputs:
       cpp_changes: ${{ steps.filter.outputs.cpp_changes }}
     steps:
-    - uses: actions/checkout@v4
-      with:
-        ref: ${{ github.event_name == 'pull_request' && github.event.pull_request.head.ref || '' }}
-    - uses: dorny/paths-filter@v2
-      id: filter
-      with:
-        filters: |
-          cpp_changes:
-            - '**/*.hpp'
-            - '**/.cpp'
-            - '**/CMakeLists.txt'
+      - uses: actions/checkout@v4
```

Comment:
ugh… we need a formatter for this at some point. Or, you know, auto-generate it ;)